api/hardwaremanagement/v1alpha1/node_pools.go

```diff
@@ -84,7 +84,7 @@ type NodePoolStatus struct {
 // +kubebuilder:resource:shortName=np
 // +kubebuilder:object:root=true
 // +kubebuilder:subresource:status
-// +operator-sdk:csv:customresourcedefinitions:displayName="ORAN O2IMS Cluster Request",resources={{Namespace, v1}}
+// +operator-sdk:csv:customresourcedefinitions:displayName="ORAN O2IMS Provisioning Request",resources={{Namespace, v1}}
```

Comment:
I know this was wrong before, but I think the displayName should just be "NodePool" here.

Reply:
Done